torch/fx/experimental/proxy_tensor.py

```diff
@@ -2268,7 +2268,7 @@ def get_proxy_mode() -> Optional[ProxyTorchDispatchMode]:
     return pre_dispatch_mode or mode
 
 
-def handle_sym_dispatch(func: Callable[_P, R], args: _P.args, kwargs: _P.kwargs) -> R:
+def handle_sym_dispatch(func: Callable[_P, R], args: _P.args, kwargs: _P.kwargs) -> R:  # type: ignore[valid-type]
```

Comment:
Interesting. This feels like a textbook case. Does args / kwargs need to be expanded into a tuple/dict using typing_extensions expand?